src/main/scala/scalaz/effect/benchmark/service/LTextService.scala

```diff
@@ -4,18 +4,18 @@ import cats.implicits._
 import cats.effect.Effect
 import fs2.{Chunk, Stream}
 import org.http4s._
+import org.http4s.client.Client
 import org.http4s.client.blaze.Http1Client
 
-class LTextService[F[_]](implicit F: Effect[F]) {
+class LTextService[F[_]](client: Client[F])(implicit F: Effect[F]) {
   import LTextService._
 
   def fetchLargeTextFile(url: String): F[String] =
     Http1Client[F]() >>= (_.get(url)(r => r.as[String]))
 
   def fetchLargeTextFilestream: F[Stream[F, Chunk[Byte]]] =
     for {
-      req <- F.delay(Request[F](Method.GET, ltUri))
-      client <- Http1Client[F]()
+      req <- F.pure(Request[F](Method.GET, ltUri))
```

Comment:
`F.pure(Request[F](Method.GET, ltUri)) >>= (F.pure(Request[F](Method.GET, ltUri))` then, instead of the unnecessary for-comprehension

Reply:
yeah i just got lazy

Reply:
No worries lol